### PR TITLE
New package: grub-btrfs-4.3

### LINKED ISSUE #21581: Package request: grub-btrfs
Include BTRFS snapshots at boot options in GRUB menu.
https://github.com/Antynea/grub-btrfs

### PR DESCRIPTION
**Pull request by anjandev**
closes  #21581

### CODE REVIEW

File: srcpkgs/grub-btrfs/template [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Template file for 'grub-btrfs'
+pkgname=grub-btrfs
+version=4.3
+revision=1
+build_style=gnu-makefile
```
Comment (sgn) [inline srcpkgs/grub-btrfs/template:5]: This package depends on `bash` and `grub`, no?

Comment (q66) [reply srcpkgs/grub-btrfs/template:5]: `grub-utils` i would say

Comment (anjandev) [reply srcpkgs/grub-btrfs/template:5]: I added grub-utils to the depends

Comment (sgn) [reply srcpkgs/grub-btrfs/template:5]: `41_snapshots-btrfs` is written in bash?

Comment (anjandev) [reply srcpkgs/grub-btrfs/template:5]: I see. I added bash to the depends list

**Comment by q66** [comment]
ah, no, i remembered wrong; `grub` is the correct dependency, i thought the config management stuff was in `grub-utils` but it's not.

**Pull request merged**